src/main/resources/api/service_openapi.yaml

```diff
@@ -79,6 +121,34 @@ components:
                 type: array
                 items:
                   type: string
+
+    CreateResourceRequestBody:
+      type: object
+      properties:
+        resourceUid:
+          description: The UID of the resource in CRL format
+          type: object
+        labels:
+          description: The labels for the resource
+          type: object
+          additionalProperties:
+            type: string
+        timeToLive:
+          description: How long created resources should live (at least) before being cleaned up.
+          type: integer
+          format: int64
+        creationTime:
+          description: When the resource is created.
+          type: integer
+          format: int64
```

Comment:
this needs units as well. Is it minutes since the epoch, seconds since the epoch?

Reply:
A timestamp in mills?